tasks/eth/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,140 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## Nested Safe State Overrides and Changes
+
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+The `approvedHashes` mapping **key** of the `ProxyAdminOwner` that should change during the simulation is
+- Council simulation: `0xfae39fd96196a971566654dbfb0f11d7b98980213f6410051ccbf81f7481039`
+- Foundation simulation: `0x0595de264ad16565584215810251811f9e56d429c8fd67def448041126dbe920`
+
+calculated as explained in the nested validation doc:
+
+```sh
```

Comment:
here, we need to add some information about the `NESTED_HASH`. -> `0xc94a72f6ae0da3b87a2ecb6e6ce52608c8a9d722a1b5f5f46ed9cc70a54f8a03` that is the nested safe hash. 
To make sure the signers knows where to get this value can compute from his side. 

```
The value of the `NESTED_SAFE` should match the following value: 
- **nested hash**: `  0xc94a72f6ae0da3b87a2ecb6e6ce52608c8a9d722a1b5f5f46ed9cc70a54f8a03
`

![7701743c6d5fa3bfb69ca45a053ba190c9a8ac8aeaa0b3f7bedab6aeca6ac2db](https://github.com/user-attachments/assets/185924f9-9757-441e-b982-ea1bffec64f9)

Reply:
We can add that, but this states "Refer to the
[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
for the expected state overrides and changes." which already explains this. I'd prefer not to duplicate the advice.

Reply:
Yes I see, I had multiples questions about it with the recents signers. 
I think we should still be putting at least the nested hash on this same page to remove friction when signers check it.

Reply:
Makes sense, fixed here [4313294](https://github.com/ethereum-optimism/superchain-ops/pull/454/commits/4313294ed2af63f21935dc125c5132e02e7064b1)